### PR TITLE
Updated BeSman with bes publish command

### PR DESCRIPTION
**Pull request by jobyko**

### CODE REVIEW

File: src/main/bash/besman-publish.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ local playbookdir="$HOME/$BESMAN_PLAYBOOK_REPO"
+ local patchdir="$HOME/$BESMAN_PATCH_REPO"
+ local env_dir="$HOME/$BESMAN_ENV_REPO"
+ local filename=$1
+
+ 
+ if [[ ( -d "$playbookdir" ) && ( -f $playbookdir/$filename ) ]]; then
+
+         cd $playbookdir
+         __besman_gh_issue-pr $filename || return 1
```
Comment (asa1997) [inline src/main/bash/besman-publish.sh:15]: Seperate the functions for issue and pr creation

File: src/main/bash/besman-repo-helpers.sh [hunk 2]
```diff
@@ -57,4 +57,20 @@ function __besman_check_github_id
     # __besman_error_rollback "$environment"
     return 1
   fi
-}
\ No newline at end of file
+}
+
+
+function __besman_gh_issue-pr
```
Comment (asa1997) [inline src/main/bash/besman-repo-helpers.sh:63]: seperate the functions for pr and create

**Review by asa1997 — CHANGES_REQUESTED** [review]
@jobyko Please address the comments

File: src/main/bash/besman-publish.sh (hunk 1 above)
Comment (jobyko) [reply src/main/bash/besman-publish.sh]: - [ ] spitted the function into two

File: src/main/bash/besman-repo-helpers.sh (hunk 2 above)
Comment (jobyko) [reply src/main/bash/besman-repo-helpers.sh]: completed

**Pull request closed without merging**